.github/workflows/python.yml

```diff
@@ -24,6 +24,8 @@ jobs:
             args: -m not integration and not gpu
           - tools: pytest
             args: -m integration
+          - tools: pytest-xdist
+            args: -n auto
```

Comment:
```suggestion
```

Won't need this, I've updated the github action for us. https://github.com/microsoft/action-python/commit/93ece9bfa602fae237f0a864fc0496b2fd56279d